frontend/packages/dev-console/src/components/pipelineruns/logs/__tests__/MultiStreamLogs.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { LogControls } from '@console/internal/components/utils';
+import { MultiStreamLogsProps, MultiStreamLogs } from '../MultiStreamLogs';
+import Logs from '../Logs';
+import { podData } from './logs-test-data';
+
+describe('MultiStreamLogs', () => {
+  let props: MultiStreamLogsProps;
+
+  beforeEach(() => {
+    props = {
+      taskName: 'step-oc',
+      resource: podData,
+    };
+  });
+
+  it('should exist', () => {
+    const wrapper = shallow(<MultiStreamLogs {...props} />);
+    expect(wrapper.isEmptyRender()).toBe(false);
+  });
+
+  it('should not render logs when containers is not present', () => {
+    props.resource.spec.containers = [];
+    const wrapper = shallow(<MultiStreamLogs {...props} />);
+    expect(wrapper.find(Logs)).toHaveLength(0);
```

Comment:
Probably want one to test the number of containers & the length of Logs are equal. Probably will want to make all containers finished for that test to not overlap with `getRenderContainers` tests.